packages/react-text/src/components/Text/useTextClasses.ts

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { makeClasses } from '@fluentui/react-theme-provider';
+
+const GlobalClassNames = {
+  root: 'ms-Text',
+};
+
+export const useTextClasses = makeClasses({
```

Comment:
this looks great, and also, I just merged `makeVariantClasses` should we want these to be easily definable and extendable in the theme. Can sync later on this.